client/src/containers/pages/Home/Home.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { Col, Row, Button } from 'antd';
+import { Link } from 'react-router-dom';
+import { getExtraData } from '@onaio/session-reducer';
+import { connect } from 'react-redux';
+import { Store } from 'redux';
+import { Dictionary } from '@onaio/utils';
+import './Home.css';
+
+export interface HomeProps {
+  extraData: Dictionary;
+}
+
+const Home: React.FC<HomeProps> = (props: HomeProps) => {
+  const { extraData } = props;
+  const { roles } = extraData;
+  return (
+    <div className="text-center">
+      <Helmet>
+        <title>OpenSRP Web</title>
+      </Helmet>
+      <Row justify="center" className="weclome-box">
+        <Col span={6}>
+          <h3>Welcome to OpenSRP</h3>
+        </Col>
+      </Row>
+      <Row gutter={16} className="links-box">
+        {roles && roles.includes('ROLE_EDIT_KEYCLOAK_USERS') && (
+          <Col className="gutter-row" span={12}>
+            <Link to="/admin" className="admin-link">
+              <Button color="outline" className="btn-links">
+                Admin
+              </Button>
+            </Link>
+          </Col>
+        )}
+        <Col className="gutter-row" span={12}>
+          <Link to="/teams" className="admin-link">
+            <Button color="outline" className="btn-links">
+              Teams
```

Comment:
Do we need to do anything in this PR towards making it easier to handle displayed strings (i.e. that need to be translatable)?

Reply:
we will address this once https://github.com/OpenSRP/web/pull/41 is finalized